spec/sidekiq/evss/disability_compensation_form/form0781_document_upload_failure_email_spec.rb

```diff
@@ -48,13 +48,16 @@
         subject.drain
       end.to trigger_statsd_increment(
         'api.form_526.veteran_notifications.form0781_upload_failure_email.success'
-      ).and trigger_statsd_increment(
-        'silent_failure_avoided_no_confirmation',
-        tags: [
-          'service:disability-application',
-          'function:526_form_0781_failure_email_queuing'
-        ]
       )
+      # It shouldn't do this any more.
+      # It now gets a definitive answer from VA notfy in the callback
```

Comment:
Oh is this one still in here on purpose to document the old behavior? I might still remove it personally

Reply:
Same goes for the other ones below